### ISSUE TITLE
Courses that have matching labs and discussions should only be matched with each other

### ISSUE THREAD

**Issue by andrewjkim** [mDibyo/berkeley-scheduler#95]
i.e. for Physics 7B there are discussions labeled ###b, and labs labeled ###a. Generally, you enroll in only matching numbers (if you enroll in 301A, you must enroll in 301B). You should probably implement an option to force this?

**Comment by andrewjkim** [comment]
Wait jk this might already work - I was reading the wrong ccns

**Comment by mDibyo** [comment]
No, this is not implemented currently. I will look into making special rules for some classes, so that we can start enforcing such constraints.
